.github/workflows/deploy-preview.yml

```diff
@@ -0,0 +1,85 @@
+name: Deploy
+on:
+  push:
+    branches-ignore:
+      - 'main'
+    paths:
+      - 'app/**'
+      - 'docs/**'
+      - '.github/workflows/deploy*.yml'
+      - 'package.json'
+      - '.vercelignore'
```

Comment:
Are there other root paths we'd want to include here which could change the docs output?